docs/dev/setup.rst

```diff
@@ -249,10 +269,12 @@ This is a service file that works on Debian/Ubuntu (using systemd):
 The following parts might also be useful for other (production) setups, even if this service file is not used:
 
 - ``-Djava.security.egd=file:/dev/./urandom``: This is required if repositories are cloned via SSH from the VCS.
-   The default (pseudo-)random-generator ``/dev/random`` is blocking which results in very bad performance when using SSH due to lack of entropy.
+   The default (pseudo-)random-generator ``/dev/random`` is blocking which results in very bad performance when using
+   SSH due to lack of entropy.
 
 
-The file should be placed at ``/etc/systemd/system/artemis.service`` and after running ``sudo systemctl daemon-reload``, you can start the service using ``sudo service artemis start``.
+The file should be placed at ``/etc/systemd/system/artemis.service`` and after running ``sudo systemctl daemon-reload``,
+you can start the service using ``sudo service artemis start``.
```

Comment:
```suggestion
you can start the service using ``sudo systemctl start artemis``.
```
IIRC The `service` command exists only in Debian/Ubuntu for backwards compatibility reasons to the old init system. `systemctl` works too, and is guaranteed to work on all systems using `systemd`.

I’m not sure if the old start also enables starting the service again after reboot, in that case the command should be
```suggestion
you can start the service using ``sudo systemctl enable --now artemis``.
```